test/unit/models/origin_test.rb

```diff
@@ -18,19 +18,19 @@
 
 class OriginTest < ActiveSupport::TestCase
   test "FactoryGirl works" do
-    assert_difference('Origin.count') do
+    assert_difference -> {Origin.count} do
       FactoryGirl.create(:origin)
     end
   end
 
   test "latest scope includes recent tweets" do
-    assert_difference('Origin.where_latest.count') do
+    assert_difference ->{Origin.where_latest.count} do
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.